apps/src/templates/teacherNavigation/LessonResources.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import {Heading6} from '@cdo/apps/componentLibrary/typography';
+import i18n from '@cdo/locale';
+
+import ResourceRow from './ResourceRow';
+
+import styles from './lesson-materials.module.scss';
+
+type LessonResourcesProps = {
+  unitNumber: number;
+  lessonNumber: number;
+  lessonPlanUrl: string | null;
+  lessonName: string | null;
+  resources: {
+    key: string;
+    name: string;
+    url: string;
+    downloadUrl: string | null;
+    audience: string;
+    type: string;
+  }[];
+};
+
+const LessonResources: React.FC<LessonResourcesProps> = ({
+  unitNumber,
+  resources,
+  lessonNumber,
+  lessonPlanUrl,
+  lessonName,
+}) => {
+  // Note that lessonPlanUrl is not needed for student resources
```

Comment:
I thought about adding a prop `isStudentResources` to determine this, but that felt unnecessary.